README.md

```diff
@@ -170,3 +188,44 @@ To update to a specific version that is not the latest version, re-run the add c
 - This repo has tests and scripts outside the app dir
 - Scripts are copied in at runtime. The tests that run on them assume they do not live inside app/.
 - Integration tests are copied into the docker image.
+
+## UPDATED - Running and testing locally
+
+Platform is now set in all the docker images but to ensure you are building for the correct platform.
+
+```
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
+```
+
+### Fastq processing
+
+Sars-CoV-2 can be built using
+
+```
+make build_sars_cov_2_local
+```
+
+There is a command to enter a shell with app mounted to /app.
+
+```
+make mounted_shell_local
+```
+
+Once inside the shell the updated pipeline can be run using
+
+```
+seqt="ont"
+nextflow run ./fastq.nf --run 61c06b0a-e5e8-4dbf-8bb0-729cce46a223 -params-file /app/local_test/$seqt/settings.json --config-path /app/local_test/$seqt/ --output_path /app/output/$seqt/
+```
+
+Note, set seqt to "illumina" to run illumina.
+
+### NCOV
+
+```
+make build_ncov_local
```

Comment:
When I run this I get an error on the step:

```
COPY docker/sars_cov_2/ncov2019-artic-nf/environments/nanopore/environment.yml /environment.yml
```

I think I need to follow the steps in `docker/README` but these don't work. `fetch_primers.py` has moved since the README was written. Can whatever needs doing be added to the Makefile?

Reply:
I needed to update the submodules. The README in the docker directory should probably be deleted.